stumpy/scrump.py

```diff
@@ -114,34 +133,61 @@ def _get_orders_ranges(n_split, m, n, orders, start, percentage):
             range_start_idx = order_idx + 1
             ranges_idx += 1
             n_dist_computed = 0
-    # Handle final range outside of for loop
-    orders_ranges[ranges_idx, 0] = range_start_idx
-    orders_ranges[ranges_idx, 1] = max_order_idx
+
+    # Handle final range outside of for loop if it was not saturated
+    if ranges_idx < orders_ranges.shape[0]:
+        orders_ranges[ranges_idx, 0] = range_start_idx
+        orders_ranges[ranges_idx, 1] = max_order_idx
 
     return orders_ranges
 
 
 @njit(fastmath=True)
 def _compute_diagonal(
-    T, m, μ, σ, orders, orders_start_idx, orders_stop_idx, thread_idx, P, I
+    T_A,
+    T_B,
+    m,
+    M_T,
+    Σ_T,
+    μ_Q,
+    σ_Q,
+    orders,
+    orders_start_idx,
+    orders_stop_idx,
+    thread_idx,
+    P,
+    I,
+    ignore_trivial,
 ):
     """
     Compute (Numba JIT-compiled) and update P, I along a single diagonal using a single
     thread and avoiding race conditions
 
     Parameters
     ----------
-    T : ndarray
+    T_A : ndarray
         The time series or sequence for which to compute the matrix profile
 
+    T_B : ndarray
+        The time series or sequence that contain your query subsequences
+        of interest
+
     m : int
         Window size
 
-    μ : ndarray
-        Sliding window mean for T
+    M_T : ndarray
+        Sliding mean of time series, `T`
 
-    σ : ndarray
-        Sliding window standard deviation for T
+    Σ_T : ndarray
+        Sliding standard deviation of time series, `T`μ_Q : ndarray
+        Mean of the query sequence, `Q`, relative to the current sliding window
```

Comment:
Looks like "μ_Q : ndarray Mean of the query sequence, `Q`, relative to the current sliding window" can be removed here